ksql-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlClientUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.client;
+
+import io.confluent.ksql.rest.Errors;
+import io.confluent.ksql.rest.entity.KsqlErrorMessage;
+import java.util.Optional;
+import java.util.function.Function;
+import javax.naming.AuthenticationException;
+import javax.ws.rs.core.Response;
+import org.eclipse.jetty.http.HttpStatus;
+import org.eclipse.jetty.http.HttpStatus.Code;
+
+public final class KsqlClientUtil {
+
+  private KsqlClientUtil() {
+  }
+
+  public static <T> RestResponse<T> toRestResponse(
```

Comment:
The contents of this new util file were lifted from `KsqlTarget`, so the methods may be accessible to the new `ServerInternalKsqlClient` as well.